tests/plugins/colordialog/manual/addhash.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: tc, feature, colordialog, 565, 4.8.0
+@bender-ui: collapsed
+@bender-ckeditor-plugins: table,tabletools,colordialog,toolbar,wysiwygarea,sourcearea,contextmenu
+
+1. Right click into table cell.
+1. Select `Cell -> Cell Properties`.
+1. Choose background color.
+1. Type color in hexadecimal 6-digit format **without** leading `#`. E.g. (`ABC123`).
+1. Confirm it.
+1. Inseted color should contain additional `#` at the beginning.
+1. Repeat above steps for color with hexadecimal 3-digits format. E.g. (`FFF`).
+
+Note: You can also check some wrong situations.
```

Comment:
It may be not clear what is **wrong situation**. You could rephrase it like:
> You can also check with colors in different formats (rgb, text) or invalid hex strings. For such colors # should not be added.